README.md

```diff
@@ -39,13 +39,13 @@ There are a number of optional inputs that can be used in the `with:` block.
 
 **tfsec_version** - the version of tfsec to use, defaults to `latest`
```

Comment:
tfsec check aws-ecr-enable-image-scans failed. 

Description: Image scanning is not enabled.

Severity: HIGH

For more information, see:

- https://aquasecurity.github.io/tfsec/v1.19.1/checks/aws/ecr/enable-image-scans/
- https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecr_repository#image_scanning_configuration